README.md

```diff
@@ -138,7 +139,13 @@ A. Put them in the `completions` subdir of `$BASH_COMPLETION_USER_DIR`
    completion code for this package. Where should I put it to be sure
    that interactive bash shells will find it and source it?**
 
-A. Install it in one of the directories pointed to by
+A. [ Disclaimer: Here, how to make the completion code visible to
+   bash-completion is explained.  We do not require always making the
+   completion code to bash-completion.  In what condition the completion code
```

Comment:
I'm having a bit of a problem understanding "We do not require always making the completion code to bash-completion". Does it mean we don't require making the completion *visible* to bash-completion?

I'm also not sure I understand the point of the disclaimer -- surely if a maintainer doesn't feel like it, they won't install their completion automatically but just include it in their tarball or similar, no matter what we say?

Reply:
> I'm having a bit of a problem understanding "We do not require always making the completion code to bash-completion". Does it mean we don't require making the completion _visible_ to bash-completion?

Ah, yes. Exactly I wanted to write *visible*. Thank you for pointing it out. I have fixed it in d4ade5b9.

> I'm also not sure I understand the point of the disclaimer -- surely if a maintainer doesn't feel like it, they won't install their completion automatically but just include it in their tarball or similar, no matter what we say?

I feel so, but a user got confused by the Q&A as reported in #695, which initially motivated this PR.